source/api/presentation/3.0/index.md

```diff
@@ -811,6 +811,41 @@ Implementations _SHOULD_ be prepared to recognize the `@id` and `@type` property
 }
 ```
 
+
+##### services
+
+A list of one or more service definitions on the top-most resource of the document, that are typically shared by more than one subsequent resource. This allows for these shared services to be collected together in a single place, rather than either having their information duplicated potentially many times throughout the document, or requiring a consuming client to traverse the entire document structure to find the information. The resource that the service applies to _MUST_ still have the `service` property, as described above, where the service resources have at least the `id` and `type` or `@id` and `@type` properties. This allows the client to know that the service applies to that resource. Usage of the `services` property is at the discretion of the publishing system.
+
+A client encountering a `service` property where the definition consists only of an `id` and `type` _SHOULD_ then check the `services` property on the top-most resource for an expanded definition.  If the service is not present in the `services` list, and the client requires more information in order to use the service, then it _SHOULD_ dereference the `id` (or `@id`) of the service in order to retrieve a service description. 
+
+The value _MUST_ be an array of JSON objects. Each object _MUST_ a service resource, as described above. 
```

Comment:
Isn't there a verb missing in the second sentence?

Reply:
Agreed --> _"... MUST **be** a service resource."_